### PR TITLE
Tests: Skip the "jQuery.ajax() on unload" test in Safari

### PR DESCRIPTION
**Pull request by mgol**
### Summary ###

The test has been already skipped in Chrome as it dropped support for such
requests and now Safari has joined the squad.

This will resolve AJAX test errors we've had for a while in Safari 13 & iOS 13.

### Checklist ###

* [x] All authors have signed the CLA at https://cla.js.foundation/jquery/jquery
* ~~New tests have been added to show the fix or feature works~~
* [x] Grunt build and unit tests pass locally with these changes
* ~~If needed, a docs issue/PR was created at https://github.com/jquery/api.jquery.com~~

### CODE REVIEW

**Comment by mgol** [comment]
Travis fails because of a bug that's fixed in #4778. I'll rebase the PR when that lands & will make sure Travis passes before landing.

File: test/unit/ajax.js [hunk 1]
```diff
@@ -2280,7 +2280,8 @@ if ( typeof window.ArrayBuffer === "undefined" || typeof new XMLHttpRequest().re
 	// Chrome 78 dropped support for synchronous XHR requests inside of
 	// beforeunload, unload, pagehide, and visibilitychange event handlers.
 	// See https://bugs.chromium.org/p/chromium/issues/detail?id=952452
-	if ( !/chrome/i.test( navigator.userAgent ) ) {
+	// Safari 13 did similar changes. The below check will catch them both.
+	if ( !/safari/i.test( navigator.userAgent ) || /edge\//i.test( navigator.userAgent ) ) {
```
Comment (mgol) [inline test/unit/ajax.js:2284]: Note to self: add a note that `edge` is needed to not exclude the test in Edge Legacy.

**Comment by mgol** [comment]
Landed on `master` in c18dc49699bc27481a4af36ed1a0ee1b19c6eb03 & on `3.x-stable` in 4f016c642cb7521d5df7e35058975353b58095fc.
